packages/react-native-performance/plugin/src/index.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  ConfigPlugin,
+  withAppDelegate,
+  withMainApplication,
+  withPlugins,
+  withProjectBuildGradle,
+} from '@expo/config-plugins';
+import {mergeContents} from '@expo/config-plugins/build/utils/generateCode';
+import {addImports} from '@expo/config-plugins/build/android/codeMod';
+
+const MATCH_DID_FINISH_LAUNCHING_IOS =
+  /-\s*\(BOOL\)\s*application:\s*\(UIApplication\s*\*\s*\)\s*\w+\s+didFinishLaunchingWithOptions:/g;
+
+const MATCH_ON_CREATE = /super\.onCreate\(\);/;
+const MATCH_DEPENDANCIES = /dependencies {/;
+
+const withRequiredIOSSetup: ConfigPlugin = expoConfig => {
+  const importStatement = '#import <ReactNativePerformance/ReactNativePerformance.h>';
+  const newLine = '  [ReactNativePerformance onAppStarted];\n';
+
+  return withAppDelegate(expoConfig, iosConfig => {
+    if (!['objc', 'objcpp'].includes(iosConfig.modResults.language)) {
+      throw new Error(
+        "react-native-google-cast config plugin does not support AppDelegate' that aren't Objective-C(++) yet.",
```

Comment:
```suggestion
        "react-native-performance config plugin does not support AppDelegate' that aren't Objective-C(++) yet.",
```

copy pasta error 🍝

Reply:
Thanks for pointing this out. I'll rename both errors 👍

Reply:
Thank you for your work on this, I really appreciate it.

Reply:
No problem! I am currently working with the team in charge of this to try and take the next steps to get it merged.